src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private static final int MIN_LOGIN_LENGTH = 6;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private static final int MIN_USER_AGE = 18;
+    private static final String VALID_EDGE_LOGIN = "valLog";
+    private static final String VALID_EDGE_PASSWORD = "valPas";
+    private static final int VALID_EDGE_AGE = 18;
+    private static final String TOO_SHORT_LOGIN = "abcde";
+    private static final String TOO_SHORT_PASSWORD = "12345";
+    private static final int TOO_YOUNG_AGE = 17;
+    private static RegistrationService registrationService;
+    private User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User();
+        validUser.setAge(VALID_EDGE_AGE);
+        validUser.setLogin(VALID_EDGE_LOGIN);
+        validUser.setPassword(VALID_EDGE_PASSWORD);
+    }
+
+    @Test
+    void register_validUser_ok() {
+        User actualUser = registrationService.register(validUser);
+        User expectUser = validUser;
```

Comment:
```suggestion
```

Reply:
Make sure that you remove it everywhere. Just use already created user.
Do not create redundant variables.